### PR TITLE
Update csi examples and deploy links

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: docs/alphafeatures.md [hunk 1]
```diff
@@ -423,7 +423,7 @@ The following section will give the steps to take snapshot and clone the cStor v
    ```
 3. Get the YAML for snapshot creation of a PVC using the following command:
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml
    ```
    In this example, downloaded file is saved as `snapshot.yaml`.
```
Comment (ranjithwingrider) [inline docs/alphafeatures.md:428]: downloaded file is saved as `snapshot.yaml` -> downloaded file is saved as `csi-snapshot.yaml`

File: docs/alphafeatures.md [hunk 2]
```diff
@@ -423,7 +423,7 @@ The following section will give the steps to take snapshot and clone the cStor v
    ```
 3. Get the YAML for snapshot creation of a PVC using the following command:
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml
```
Comment (ranjithwingrider) [inline docs/alphafeatures.md:426]: https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml -> https://raw.githubusercontent.com/openebs/cstor-csi/v1.9.x/examples/csi-snapshot.yaml

Comment (ranjithwingrider) [inline docs/alphafeatures.md:426]: https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml -> https://raw.githubusercontent.com/openebs/cstor-csi/v1.9.x/examples/csi-pvc-clone.yaml

File: docs/alphafeatures.md [hunk 3]
```diff
@@ -451,9 +451,9 @@ The following section will give the steps to take snapshot and clone the cStor v
    
 6. Now, let's create clone volume using the above snapshot. Get the PVC YAML spec for creating the clone volume from the given snapshot.
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/pvc-clone.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-pvc-clone.yaml
```
Comment (ranjithwingrider) [inline docs/alphafeatures.md:454]: https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-pvc-clone.yaml -> https://raw.githubusercontent.com/openebs/cstor-csi/v1.9.x/examples/csi-pvc-clone.yaml

File: docs/alphafeatures.md (hunk 1 above)
Comment (ranjithwingrider) [inline docs/alphafeatures.md:428]: In this example, downloaded file is saved as `snapshot.yaml`. -> In this example, downloaded file is saved as `csi-snapshot.yaml`.

File: docs/alphafeatures.md [hunk 4]
```diff
@@ -423,7 +423,7 @@ The following section will give the steps to take snapshot and clone the cStor v
    ```
 3. Get the YAML for snapshot creation of a PVC using the following command:
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml
    ```
    In this example, downloaded file is saved as `snapshot.yaml`.
 4. Edit the snapshot.yaml which is created in previous step to update:
```
Comment (ranjithwingrider) [inline docs/alphafeatures.md:429]: Edit the snapshot.yaml which is created in previous step to update: -> Edit the `csi-snapshot.yaml` which is created in the previous step to update:

**Comment by kmova** [comment]
@prateekpandey14 can you raise the PR again from a new branch. There was a rebase done on staging.

**Comment by kmova** [comment]
Can you please open a new PR with the changes on top of the new staging.

**Pull request closed without merging**